### PR TITLE
Local branch

### PR DESCRIPTION
**Pull request by mvheetve**
I created a branch to add data that we are using to test/validate the modules of nf-cmgg-structural.
I added crams and beds + added info about the files to the README.

M

### CODE REVIEW

**Review by matthdsm — CHANGES_REQUESTED** [review]
LGTM, but drop all the filename prefixing eg. `nf-cmgg.SVcontrol` 
Also update the config file so the data can be easily used in the workflow.

**Comment by mvheetve** [comment]
changed the names and added the files to config. Please let me know if you need anything else. 

M

**Review by matthdsm — CHANGES_REQUESTED** [review]
LGTM again, but I WOULD use a prefix in the config so it's easier to determine which file we're using. eg. `SV.PosCon...` or whatever.
Naming and config key should match nf-core testdata as close as possible
https://github.com/nf-core/test-datasets/tree/modules/data/genomics/homo_sapiens/illumina/bam

File: nf-cmgg-testdata.config [hunk 1]
```diff
@@ -23,7 +23,18 @@ params {
                 test_per_base               = "${test_data_dir}/genomics/homo_sapiens/illumina/bed/test.per-base.bed.gz"
                 test_per_base_index         = "${test_data_dir}/genomics/homo_sapiens/illumina/bed/test.per-base.bed.gz.csi"
                 roi_bed                     = "${test_data_dir}/genomics/homo_sapiens/illumina/bed/roi.bed"
-
+                Poscon1_cram                = "${test_data_dir}/genomics/homo_sapiens/illumina/cram/SVcontrol/PosCon1.cram"
```
[the remark this answers is not in the document]
Comment (mvheetve) [reply nf-cmgg-testdata.config]: Let me know how you like it now :)

**Review by matthdsm — APPROVED** [review]
Go for it!

**Pull request merged**